test/test_common/utility.h

```diff
@@ -834,6 +834,13 @@ template <class Interface, class Impl> class TestHeaderMapImplBase : public Inte
     HeaderMapImpl::copyFrom(*header_map_, rhs);
     header_map_->verifyByteSizeInternalForTest();
   }
+  void copyFrom(const TestHeaderMapImplBase& rhs) {
```

Comment:
This utility change exposes copyFrom on the underlying headermap using a similar pattern found elsewhere in the file. It allows the unit test to construct an empty header map, add single fields, and then inherit multiple fields from an existing map without too much trouble. This allows a test writer to make header map comparison code happy, since it cares about order.